src/lib.rs

```diff
@@ -74,6 +75,24 @@
 //! >   its text symbols by adding the bias to their SVMAs.
 //!
 //! [LUL]: https://searchfox.org/mozilla-central/rev/13148faaa91a1c823a7d68563d9995480e714979/tools/profiler/lul/LulMain.h#17-51
+//!
+//! ## Names and IDs
+//!
+//! `findshlibs` also gives access to module names and IDs.  Since this is also
+//! not consistent across operating systems the following general rules apply:
+//!
+//! > * `id` refers to the ID of the object file itself.  This is generally
+//! >   available on all platforms however it might still not be compiled into
+//! >   the binary in all case.  For instance on Linux the `gnu.build-id` note
+//! >   needs to be compiled in (which Rust does automatically).
+//! > * `debug_id` refers to the ID of the debug file.  This only plays a role
+//! >   on Windows where the executable and the debug file (PDB) have a different
+//! >   ID.
+//! > * `name` is the name of the executable.  On most operating systems (and
+//! >   all systems implemented currently) this is not just the name but in fact
+//! >   the entire path to the executable.
+//! > * `debug_name` refers to the debug file if it's different.  This is again
+//! >   the case on windows where this will be the path to the PDB file.
```

Comment:
On macOS debuginfo is also in a separate file most of the time. Most of the time inside a .dSYM bundle, but it can also be inside the original object files before linking. On ELF systems something similar may apply too. You need https://github.com/gimli-rs/locate-dwarf to locate it. I think this method should be renamed to `pdb_debug_file` to indicate that it only applies to pdb files and not all debuginfo on all platforms.

Reply:
I changed the first sentence